build/aks-variables.yml

```diff
@@ -0,0 +1,22 @@
+variables:
+  deploymentName: 'cd-dicom'
+  azureSubscriptionName: '10M Client Demo'
```

Comment:
@moiradillon12 do you know what we normally put as dummy variables we expect the user to override, there are a few of them in this file & pr